lib/ht_sip_validator/validator/image/filenames.rb

```diff
@@ -1,36 +0,0 @@
-# frozen_string_literal: true
-
-module HathiTrust::Validator::Image
-  class Filenames < HathiTrust::Validator::Base
-
-    def perform_validation
-      image_files = HathiTrust::Validator::Image.image_files(@sip.files)
-      return no_images_error if image_files.empty?
```

Comment:
Are other validators covering the case were a submission package comes in that is otherwise correct, but has no image files?